src/js/angular/guides/steps/complex/ttyg/create-ttyg-agent/plugin.js

```diff
@@ -5,16 +5,8 @@ PluginRegistry.add('guide.step', [
             const GuideUtils = services.GuideUtils;
             options.mainAction = 'create-ttyg-agent';
 
-            const configureExtractionMethods = (services, options) => {
-                const methods = options.methods || [];
-
-                return methods.map((method) => {
-                    return {
-                        guideBlockName: method.guideBlockName,
-                        options: method.options
-                    };
-                });
-            };
+            // Set name field as mandatory for creation
+            options.name = true;
```

Comment:
Don't do this here. The configuration should only be on the 'configure-agent' step:
```
{
    guideBlockName: 'configure-agent',
    options: angular.extend({}, options, {editName: true})
}

```

Reply:
Passed only to the `configure-agent`